lib/widgets/list_of_cards.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+import 'package:hearthstoneapp/src/model/hearthstone_card.dart';
+import 'package:hearthstoneapp/src/repository/card_list_repository.dart';
+import 'package:hearthstoneapp/utils/constants.dart';
+import 'package:hearthstoneapp/utils/palette.dart';
+import 'package:hearthstoneapp/widgets/card.dart';
+
+class ListOfCards extends StatelessWidget {
+  ListOfCards({Key? key}) : super(key: key);
+  final CardListRepository repo = CardListRepository();
+  late final Future<List<HearthstoneCard>> listOfCards;
+
+  @override
+  Widget build(BuildContext context) {
+    listOfCards = repo.readJson(Constants.jsonPath);
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Palette.boxColor,
+        title: Constants.listViewTitle,
+      ),
+      body: FutureBuilder(
+        future: listOfCards,
+        builder: (
+          context,
+          AsyncSnapshot<List<HearthstoneCard>> snapshot,
+        ) {
+          List<Widget> children = [];
+          if (snapshot.hasData) {
+            snapshot.data?.forEach((element) {
```

Comment:
You know that snapshot.data gave you a List<HearthstoneCard> so instead to name inside for each an 'element' you can name it hearthstoneCard

Reply:
Done!